pkg/rest/coverage/swagger_analysis.go

```diff
@@ -0,0 +1,119 @@
+package coverage
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-openapi/loads"
+	"github.com/go-openapi/spec"
+)
+
+var refCache = make(map[string]int)
+
+type RequestStats struct {
+	Body         map[string]int
+	Query        map[string]int
+	ParamsHit    int
+	ParamsNum    int
+	MethodCalled bool
+	Path         string
+	Method       string
+}
+
+func getRESTApiStats(swaggerPath string, filter string) (map[string]map[string]*RequestStats, error) {
```

Comment:
Would you please add explanation comment for each function here doing low level stuff here? It will simplify the future work. Thanks.